test/common/module_api.test.lua

```diff
@@ -6,12 +6,36 @@ package.cpath = './?.so;' .. package.cpath
 local memcached = require('memcached')
 local test = tap.test('memcached module api')
 
-local function is_port_open(port)
-    local sock, _ = socket.tcp_connect('127.0.0.1', port)
-    if not sock then
+-- 1. Open connection to memcached instance.
```

Comment:
Nit: perhaps this should be squashed with efe54476?

Reply:
No. First version is a simple check port for openness. After introducing own small arena I need a simple memcached client to trigger allocations in memcached fiber.